test/scripts/setup-services-run-tests.sh

```diff
@@ -145,6 +143,10 @@ kubectl -n $NAMESPACE port-forward $TARGET_GRPC_POD 8081:8080 &
 # Stream server logs.
 kubectl -n $NAMESPACE logs -f $TARGET_GRPC_POD &
 
+# Run Python tests
+pip install ml-metadata==0.15.0
```

Comment:
Is that better ">=" to be consistent with REQUIRES so catch the problem if new ml-metadata introduces?

Reply:
Good suggestion. I change setup.py to use `==`.